src/libraries/System.IO.Compression/src/System/IO/Compression/PositionPreservingWriteOnlyStreamWrapper.cs

```diff
@@ -42,7 +42,7 @@ public override void Write(ReadOnlySpan<byte> buffer)
             _stream.Write(buffer);
         }
 
-        public override IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback? callback, object? state)
+        public override IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback callback, object? state)
```

Comment:
Was this change intentional?

Reply:
whoops not intended, just for local build, will revert